client/app/visualizations/chart/index.js

```diff
@@ -53,9 +56,26 @@ function ChartEditor(ColorPalette, clientConfig) {
 
       scope.stackingOptions = {
         Disabled: null,
-        Enabled: 'normal',
-        Percent: 'percent',
+        Stack: 'stack',
+        Overlay: 'overlay',
+        Group: 'group',
+        Relative: 'relative',
       };
+      scope.allowedStackingOptions = [];
+      scope.isStackingOptionAllowed = value => scope.allowedStackingOptions.indexOf(value.value) >= 0;
+
+      scope.$watch('options.globalSeriesType', () => {
```

Comment:
Can we use `ng-change` on the input field instead of `$watch`?

Reply:
Okay, will do